tests/unit/index.ts

```diff
@@ -60,6 +61,44 @@ registerSuite({
 			assert.notInclude(contents, `module 'foo/index'`);
 		});
 	},
+	'project with outDir and rootDir - directory handling stress test': function () {
```

Comment:
So the history on your branch shows it being added in the commit after https://github.com/dgoldstein0/dts-generator/commit/9b1ad54949607594dae778f88e6ac1be42aa8acd , which is commit https://github.com/dgoldstein0/dts-generator/commit/cc49f32bc60c3a114a9f65f4ebf1dab9c20a94c6, which is where it was also added in master, e.g. cc49f32. Either way, I think it will be fine, so I'll go ahead and land this.